### ISSUE TITLE
Add a `.cpp` implementation for `Huffmann` compression algorithm.

### ISSUE THREAD

**Issue by Ashrockzzz2003** [Ashrockzzz2003/Data_Structures_and_Algorithms#5]
- [ ] Add a `.cpp` implementation for `Huffmann` compression algorithm.

**Comment by WinterSun23** [comment]
@Ashrockzzz2003 I have some experience working in cpp. Can you assign this issue to me?

**Comment by Ashrockzzz2003** [comment]
@WinterSun23 Sure. But do remember you can only work on 2 issues at the same time. I see that you've already been assigned to another issue in Match-da-pairs repo. Are you still willing to contribute? 
This means, only after you finish one of the two, you'll be assigned to another issue.

**Comment by WinterSun23** [comment]
Yes I am still willing to work on this.

**Comment by Ashrockzzz2003** [comment]
Assigned you to this.
You've time until `Jan 2, 2025` to raise a PR on this.
